### PR TITLE
chore(format):  clean up some unused code.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

part of #7732

### CODE REVIEW

**Review by Xuanwo — APPROVED** [review]

**Comment by fkuner** [comment]
Is the format diagnosis no longer used?

**Review by BohuTANG — APPROVED** [review]

**Comment by youngsofun** [comment]
> Is the format diagnosis no longer used?

yes, it was used mainly for csv/tsv

new impls cut file to bytes of rows and fields first,  then interpreter bytes of each field according to datatype，
the error type and err context is already clear enough, and more easier to get the error data (no need checkpoint).

File: src/query/service/src/pipelines/executor/executor_graph.rs [hunk 1]
```diff
@@ -241,8 +241,15 @@ impl ExecutingGraph {
                 if state_guard_cache.is_none() {
                     state_guard_cache = Some(node.state.lock().unwrap());
                 }
-
-                let processor_state = match node.processor.event()? {
+                let event = node.processor.event()?;
+                tracing::debug!(
```
Comment (sundy-li) [inline src/query/service/src/pipelines/executor/executor_graph.rs:245]: we should use trace level here.

Comment (youngsofun) [reply src/query/service/src/pipelines/executor/executor_graph.rs:245]: fixed in https://github.com/datafuselabs/databend/pull/7927

**Pull request merged**